download.md

```diff
@@ -38,31 +42,30 @@ This is the initial release of PELUX.
 
 ### Artifacts
 * These release notes.
-* [pelux.xml](https://github.com/Pelagicore/pelux-manifests/blob/v1.0/pyro/pelux.xml)
+* [pelux.xml](https://github.com/Pelagicore/pelux-manifests/blob/v2.0/rocko/pelux.xml)
   in PELUX Yocto Manifests repository, see below.
 * The version of [PELUX Software Development
-  Environment](https://github.com/Pelagicore/pelux-sde/tree/v1.0) as pointed out
+  Environment](https://github.com/Pelagicore/pelux-sde/tree/v2.0) as pointed out
   below.
-* The version of [PELUX Software Factory Baseline](//pelux.io/software-factory/v1.0/) as
+* The version of [PELUX Software Factory Baseline](//pelux.io/software-factory/v2.0/) as
   pointed out below.
 
 ### Branches and tags
 * PELUX Yocto Manifests
     - URL: [https://github.com/Pelagicore/pelux-manifests](https://github.com/Pelagicore/pelux-manifests)
-    - branch morty, tag v1.0/morty
-    - branch pyro, tag v1.0/pyro
+    - branch rocko, tag v2.0/rocko
     - The same branch and tag scheme is used for the internal layers
       meta-bistro, meta-pelux and meta-template
 
 * PELUX Software Development Environment
     - URL: [https://github.com/Pelagicore/pelux-sde/](https://github.com/Pelagicore/pelux-sde/)
-    - branch PELUX-1.0
-    - tag v1.0
+    - branch PELUX-2.0
+    - tag v2.0
 
 * PELUX Software Factory Baseline
     - URL: [https://github.com/Pelagicore/software-factory](https://github.com/Pelagicore/software-factory)
-    - branch PELUX-1.0
-    - tag v1.0
+    - branch PELUX-2.0
+    - tag v2.0
 
 ------------------------
```

Comment:
Maybe we want to keep some information about Pelux 1.0. And write that it is not updated. Just so we don't loose that information completely, I mean.

Reply:
Done.